main.go

```diff
@@ -3,13 +3,42 @@ package main
 import (
 	"fmt"
 	"net/http"
-
+	"log"
+	"encoding/json"
+	"github.com/spf13/viper"
 	"github.com/gorilla/mux"
 )
 
+type infoResponse struct{
+    Name string `json:"name"`
+    AppVersion string `json:"appVersion"`
+}
+
+type healthResponse struct{
+    Database string `json:"database"`
+    Rabbitmq string `json:"rabbitmq"`
+}
+
+func showInfo(w http.ResponseWriter, r *http.Request) {
+    vi := viper.New()
```

Comment:
Viper should be set up in main() and can be set up using `viper.AutomaticEnv()` (as all of the stuff in Chart.yaml gets put into env variables) - you can then call `viper.GetString('key')` from anywhere and don't need to create an actual instance of viper or read in config or anything.

Would suggest deleting these 3 lines and just adding `viper.AutomaticEnv()` at the top of main()